optimade/models/jsonapi.py

```diff
@@ -55,16 +56,39 @@ class JsonApi(BaseModel):
 class ToplevelLinks(BaseModel):
     """A set of Links objects, possibly including pagination"""
 
+    class Config:
+        extra = "allow"
```

Comment:
Minor thing, but could this be moved to the bottom of the class to align with all other uses of `Config` (that I know of)?